linode/instance/resource.go

```diff
@@ -830,6 +830,11 @@ func updateResource(ctx context.Context, d *schema.ResourceData, meta interface{
 		}
 	}
 
+	if d.HasChange("ipv4") {
+		// Log a warning that direct IP changes are not supported
+		return diag.Errorf("Changing IPv4 addresses after instance creation is not directly supported. The attempted change will be ignored.")
+	}
+
```

Comment:
Could this be removed since the `ipv4` field was made `ForceNew` in a previous PR?

Reply:
Yeah makes sense! I will be removing it in the next commit